### DISCUSSION TITLE
Ring Camera Light Sensor Information

### DISCUSSION THREAD

**Discussion by mirkokg** [tsightler/ring-mqtt#823] [Feature Requests]
I have no idea if its possible, can you add light sensor information, (luminance in lux). This would be very useful for automations.

I would like to turn on the light in combination with motion detection and if light sensor detects that its too dark. Currently I have to use separate external light sensor to do this, its pity because camera has internal light sensor but its values are not exposed via ring2mqtt.

**Comment by tsightler** [comment]
I think this falls into the 2nd question in the FAQ, i.e. "Is it possible to perform random thing not possible in the Ring app".  The general answer is, if the capability isn't exposed somewhere in the Ring app, then it's probably not possible.  This isn't 100% true, but about 99% this is the case.  As far as I know this luminance information is not available via the Ring API, it is used only internally by the camera firmware.  If you can prove differently I would be willing to consider exposing it, but I've never seen anything that appears to be this value in the data output.

**Reply by mirkokg** [reply]
@tsightler I'll look into it, but I think you are right. :-(

**Reply by tsightler** [reply]
I took a quick glance through the API output from my cameras, and I didn't see anything in it that appears to be this data.
